source/includes/_services.md

```diff
@@ -45,6 +45,159 @@ entity | yes | One of: "Candidate", "ClientContact", or "Lead".
 ids | yes | List of IDs of the given type of entity, a maximum of 500 per call.
 BhRestToken | no | Token that represents a session established by the login process. Must be sent with all subsequent requests to the API. The session key can be provided in the BhRestToken query string, a cookie, or an HTTP header.
 
+## POST / PUT /services/CorporateUser
+
+With appropriate access, you can add or update users. Adding users may incur additional user fees. For questions about your account and billing, please contact Support or your Account Manager.
```

Comment:
Maybe use "Bullhorn technical support or your account manager" with lower case as here.

Reply:
Changed Support to Bullhorn Support to match verbiage used in client facing documentation.
![image](https://github.com/user-attachments/assets/1af53803-c461-487d-b968-c28fe44e8303)